edgePy/util.py

```diff
@@ -0,0 +1,23 @@
+""" Utilities to support functions and classes """
+import logzero  # type: ignore
+
+
+def _setup_log():
+    """Formats and sets up the logger instance.
+
+    Returns:
+        An instance of a logger.
+    """
+    _log_format = (
```

Comment:
No need to privatize or hardcode. Allow this function to override by passing in `formatter` as a parameter.

Reply:
What I mean here is something like:


```python
def function(param: Optional[Formatter] = None):
    param = param or logzero.LogFormatter(...)
```

Reply:
Thanks for the clarification here, @clintval! Definitely more logical.